### PR TITLE
solved id issues and timeout issue

### PR DESCRIPTION
**Pull request by ankitmashu**

### CODE REVIEW

**Review by kailash — APPROVED** [review]

**Comment by kailash** [comment]
ok to test

**Comment by karun-singh** [comment]
retest this please

**Comment by karun-singh** [comment]
retest this please

**Comment by kailash** [comment]
retest this please

File: src/test/java/iudx/resource/server/database/archives/DatabaseServiceTest.java [hunk 1]
```diff
@@ -2101,7 +2101,9 @@ void searchAttribute3(VertxTestContext testContext) {
         .search(request)
         .onFailure(
             handler -> {
-              assertEquals(handler.getMessage(),"Result Limit exceeds");
+              assertEquals(
+                  handler.getMessage(),
+                  "{\"type\":413,\"title\":\"urn:dx:rs:payloadTooLarge\",\"details\":\"Response size exceeds limit\"}");
```
Comment (kailash) [inline src/test/java/iudx/resource/server/database/archives/DatabaseServiceTest.java:2106]: Create a JsonObject instead of a String.

**Review by kailash — CHANGES_REQUESTED** [review]

File: src/test/java/iudx/resource/server/database/archives/DatabaseServiceTest.java (hunk 1 above)
Comment (ankitmashu) [reply src/test/java/iudx/resource/server/database/archives/DatabaseServiceTest.java]: Solved

**Comment by kailash** [comment]
retest this please

**Comment by kailash** [comment]
retest this please

**Review by kailash — APPROVED** [review]

**Pull request merged**